util/buildRelease/install.sh

```diff
@@ -121,12 +121,26 @@ then
   DEST_THIRD_PARTY="$PREFIX/lib/chapel/$VERS/third-party"
   DEST_CHPL_HOME="$PREFIX/share/chapel/$VERS"
   echo "Installing Chapel split to bin, lib, share to $PREFIX"
+  if [ "$CHPL_HOME" = "$PREFIX" ]
```

Comment:
Look at http://tldp.org/LDP/abs/html/comparison-ops.html and search for "string comparison"